src/prelude/prelude.ml

```diff
@@ -1,3 +1,13 @@
 let prelude = [%blob "prelude.mo"]
-let internals = [%blob "internals.mo"]
-let prim_module = [%blob "prim.mo"]
+let internals' = [%blob "internals.mo"]
+let timers = [%blob "timers.mo"]
+let internals want_timers =
+  if want_timers
+  then internals' ^ timers
```

Comment:
Even better, abstract out the timer stuff into a base library if possible. That might also make it easier to test, though I'm not sure about that since you also depend on Time.now as well as ic0.global_timer_set

Reply:
Happy to do it if I figure out how to reliably call a function that resides in some user-defined location from `desugar.ml` or `compile.ml`. Currently I can't see a way...